src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/LineWrappingHandler.java

```diff
@@ -69,6 +69,16 @@ public static LineWrappingOptions ofBoolean(boolean val) {
 
     }
 
+    /**
+     * The list of ignored token types for being checked by lineWrapping indentation
+     * inside {@code checkIndentation()}.
```

Comment:
is it because they have their own handlers ?

Reply:
Yes, they do interuptions with the dedicated handlers, like here in this issue.
So if everything is covered inside the handler, then no need to explicitly check it against the lineWrappingHandler.

Reply:
please mention a reason in javadoc, to be more explicit

Reply:
done

Reply:
please make javadoc links to such classes, wedo not have that much of them, it make sense to reference.

Reply:
Done